src/routes/index.ts

```diff
@@ -1,13 +1,33 @@
-import { Router } from "express";
+import Router, {Request, Response} from "express"
 import prisma from "../prisma";
+import auth from "./auth";
+import signup from "./signup";
+import course from "./course";
 
 const routes = Router();
 
-routes.get("/", (_, res) => {
-    console.log("connect");
+// enables passing json bodies.
+routes.use(Router.json());
+
+routes.use("/course", course);
+routes.use("/login", auth);
+routes.use("/login/sign-up", signup);
```

Comment:
høre sign-up til under login? hierarkisk giver det ik mening for mig, synes det skal være sin egen